### ISSUE TITLE
IdeaSideException/TimeoutException while working with the gradle tasks tree (JTreeFixture) on OS X

### ISSUE THREAD

**Issue by zcervink** [JetBrains/intellij-ui-test-robot#199]
- issue appear only on OS X
- issue reported and described here (there is a workaround - use the button for expanding the gradle tasks tree) - https://github.com/redhat-developer/intellij-common-ui-test-library/issues/126

- The following method call caused IdeaSideException/TimeoutException while working with the gradle task tree:
`gradleTaskTree().expandAll();`

- gradleTaskTree() is JTreeFixture for the gradle tasks tree:
`find(JTreeFixture.class, JTreeFixture.Companion.byType(), Duration.ofSeconds(10));`

- [LINK TO THE CODE WHERE EXCEPTION APPERS](https://github.com/redhat-developer/intellij-common-ui-test-library/blob/main/src/main/java/com/redhat/devtools/intellij/commonuitest/fixtures/mainidewindow/toolwindowspane/buildtoolpane/GradleBuildToolPane.java#L76)

- if it is an issue in the Remote-Robot it should be fixed

### Stacktrace:
```
com.intellij.remoterobot.client.IdeaSideException: Wrapped java.util.concurrent.TimeoutException
	at com.intellij.remoterobot.client.IdeRobotClient.throwIdeaSideError(IdeRobotClient.kt:142)
	at com.intellij.remoterobot.client.IdeRobotClient.processExecuteResponse(IdeRobotClient.kt:109)
	at com.intellij.remoterobot.client.IdeRobotClient.execute(IdeRobotClient.kt:63)
	at com.intellij.remoterobot.JavaScriptApi$DefaultImpls.runJs(JavaScriptApi.kt:57)
	at com.intellij.remoterobot.RemoteRobot.runJs(RemoteRobot.kt:32)
	at com.intellij.remoterobot.fixtures.Fixture.runJs(Fixture.kt:92)
	at com.intellij.remoterobot.fixtures.Fixture.runJs$default(Fixture.kt:85)
	at com.intellij.remoterobot.fixtures.JTreeFixture$expandAll$1.invoke(JTreeFixture.kt:132)
	at com.intellij.remoterobot.fixtures.JTreeFixture$expandAll$1.invoke(JTreeFixture.kt:131)
	at com.intellij.remoterobot.stepsProcessing.StepWorkerKt.step(StepWorker.kt:23)
	at com.intellij.remoterobot.fixtures.JTreeFixture.expandAll(JTreeFixture.kt:131)
	at com.redhat.devtools.intellij.commonuitest.fixtures.mainidewindow.toolwindowspane.buildtoolpane.GradleBuildToolPane.buildProject(GradleBuildToolPane.java:76)
```

**Comment by zcervink** [comment]
@nizienko thank you very much for you advice. I have tried that, but it does not work (same issue on OS X):
https://github.com/redhat-developer/intellij-quarkus/runs/7327241383?check_suite_focus=true

And I do not think that I found a wrong JTree - my implementation works for Windows and Linux and on OS X on localhost, but not OS X on github actions. And when I use your code, it is exactly the same situation. And I am using the same tree fixture on the [**next line**](https://github.com/redhat-developer/intellij-common-ui-test-library/blob/main/src/main/java/com/redhat/devtools/intellij/commonuitest/fixtures/mainidewindow/toolwindowspane/buildtoolpane/GradleBuildToolPane.java#L77) to find the build item in that tree and this line of code works as expected (with workaround for previous line). I tried also adding `gradleTaskTree().clickRow(0);` before `expandAll()`, this line works, so the right tree has been found.

Maybe it is not an issue in Remote-Robot and maybe it is not an issue in addressing the JTree. Maybe just the IntelliJ on some versions of OS X throws some IdeaSideExceptions. At least the workaround exist - there is a button above the tree that has the 'expand the tree' functionality.
